presto-main/src/test/java/com/facebook/presto/sql/TestExpressionsAndTupleDomainFilters.java

```diff
@@ -112,7 +112,8 @@
 import static org.testng.Assert.assertEquals;
 import static org.testng.Assert.assertTrue;
 
-public class TestTupleDomainFilterUtils
+public class TestExpressionsAndTupleDomainFilters
```

Comment:
What about moving the tests to presto-tests instead?

Reply:
Done